.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,25 @@
+name: Trivy Scanner - EBS CSI Driver
+on:
+  pull-request:
+  
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          ignore-unfixed: true
+          format: 'sarif'
+          output: 'trivy-ebs-csi-results.sarif'
```

Comment:
Probably just call this `results.sarif` to be consistent with the other workflow?